ferrocene/doc/internal-procedures/src/working-with-the-ci.rst

```diff
@@ -188,6 +183,39 @@ of the job:
    ./x clean
    ./x --stage 2 dist rust-std
 
+Preserving build artifacts
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you wish to preserve your ``build/`` artifacts, it may make sense to
+re-clone the Ferrocene repository inside the container or mount a dedicated
+folder for ``/ferrocene/build``:
+
+.. code-block:: bash
+
+   docker run --rm --tty --interactive --workdir /ferrocene \
+      --mount "type=bind,src=$(pwd),dst=/ferrocene" \
+      --mount "type=bind,src=$(pwd)/container-build,dst=/ferrocene/build" \
+      ubuntu-20 bash
+
+Note that you will need to create the `container-build` folder beforehand.
```

Comment:
should specify how to make build system use container-build/ instead of build/

Reply:
you don't need to do anything for that, the container will see `container-build` as `build`.

Reply:
oh, indeed!